app/options-toolbar/options-toolbar.html

```diff
@@ -6,7 +6,7 @@
 
 </style>
 
-<options-toolbar change.setvalue="myObj.state = prop(this, 'dataset.value')">
+<options-toolbar data-value="off" change.setvalue="myObj.state = prop(this, 'dataset.value')">
```

Comment:
We do not want it driven by the attribute. Lets discuss when you online

Reply:
use aria-selected on button instead and select that

Reply:
use process-api set-selection added by charles